src/code/z_actor.c

```diff
@@ -97,23 +132,44 @@ GLOBAL_ASM("asm/non_matchings/z_actor//func_800B6608.asm")
 
 GLOBAL_ASM("asm/non_matchings/z_actor//func_800B6680.asm")
 
-GLOBAL_ASM("asm/non_matchings/z_actor//Actor_MarkForDeath.asm")
+void Actor_MarkForDeath(Actor* actor) {
+    actor->draw = NULL;
+    actor->main = NULL;
+    actor->flags &= -2;
+}
 
-GLOBAL_ASM("asm/non_matchings/z_actor//Actor_InitCurrPosition.asm")
+void Actor_InitCurrPosition(Actor* actor) {
+    actor->currPosRot = actor->initPosRot;
+}
 
 GLOBAL_ASM("asm/non_matchings/z_actor//Actor_SetHeight.asm")
 
-GLOBAL_ASM("asm/non_matchings/z_actor//Actor_SetRotationFromDrawRotation.asm")
+void Actor_SetRotationFromDrawRotation(Actor* actor) {
+    actor->currPosRot.rot = actor->drawParams.rot;
+}
 
-GLOBAL_ASM("asm/non_matchings/z_actor//Actor_InitDrawRotation.asm")
+void Actor_InitDrawRotation(Actor* actor) {
+    actor->drawParams.rot = actor->currPosRot.rot;
+}
 
-GLOBAL_ASM("asm/non_matchings/z_actor//Actor_SetScale.asm")
+void Actor_SetScale(Actor* actor, f32 scale) {
+    actor->scale.z = scale;
+    actor->scale.y = scale;
+    actor->scale.x = scale;
+}
 
 GLOBAL_ASM("asm/non_matchings/z_actor//Actor_SetObjectSegment.asm")
 
 GLOBAL_ASM("asm/non_matchings/z_actor//Actor_InitToDefaultValues.asm")
 
-GLOBAL_ASM("asm/non_matchings/z_actor//Actor_FiniActor.asm")
+void Actor_FiniActor(Actor* actor, GlobalContext* ctxt) {
+    if(actor->init == NULL){
+        if(actor->fini != NULL){
```

Comment:
In Majora's Mask we generally put spaces after the parenthesis and before the brace.

Reply:
Fixed